Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1075,6 +1077,8 @@ public extension Purchases {
                 .build())
     }
 
+    #endif
+
     /**
```

Comment:
it'd be great to disable everything else (other than this method and `switchUser`) for this mode

Reply:
Oooh, you mean the other `configure` methods, `logIn`, etc?

Reply:
Done in #2442